insQ.js

```diff
@@ -49,16 +49,23 @@ var insertionQ = (function () {
 
         document.head.appendChild(styleAnimation);
 
-        var bindAnimationLater = setTimeout(function () {
+        if(options.timeout > 0) {
```

Comment:
Create a variable for whether there is a timeout > 0 instead of writing it in 2 places. Also I would suggest creating a function to add the eventListeners so you can call it in both places instead of reusing all of the code.